src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -70,26 +71,36 @@ def get_attrs(self, item_list, user, **kwargs):
             rule_result = result[alert_rules[alert_rule_id]].setdefault("projects", [])
             rule_result.append(project_slug)
 
-        for rule_activity in AlertRuleActivity.objects.filter(
-            alert_rule__in=item_list, type=AlertRuleActivityType.CREATED.value
-        ).select_related("alert_rule", "user"):
-            if rule_activity.user:
-                user = {
-                    "id": rule_activity.user.id,
-                    "name": rule_activity.user.get_display_name(),
-                    "email": rule_activity.user.email,
-                }
-            else:
-                user = None
+        rule_activities = list(
+            AlertRuleActivity.objects.filter(
+                alert_rule__in=item_list, type=AlertRuleActivityType.CREATED.value
+            )
+        )
 
-            result[alert_rules[rule_activity.alert_rule.id]].update({"created_by": user})
+        rule_activities_by_user_id = {r.user_id: r for r in rule_activities}
```

Comment:
Is there only one activity per user? Or could a user have multiple activities?

Reply:
Good point, and I'll adjust